File: Code coach/Medium/Deja vu/deja_vu.cpp
#include <iostream>
using namespace std;

int main()
{
	string chars;
	cin >> chars;

	char previousChar = ' ';
	bool isRepeated = false;

	for (auto &ch : chars)
	{
		if (ch == previousChar)
		{
			cout << "Deja Vu";
			isRepeated = true;
			break;
		}
		else
		{
			previousChar = ch;
		}
	}

	if (!isRepeated)
	{
		cout << "Unique";
	}

	return 0;
}

File: Code coach/Medium/Deja vu/deja_vu.py
chars = input()
previous_char = None
is_repeated = False

for char in chars:
	if char is previous_char:
		print('Deja Vu')
		is_repeated = True
		break
	else:
		previous_char = char

if (not is_repeated):
	print('Unique')
